python/cuml/dask/neighbors/kneighbors_classifier.py

```diff
@@ -63,18 +59,14 @@ class KNeighborsClassifier(NearestNeighbors):
     verbose : int or boolean, default=False
         Sets logging level. It must be one of `cuml.common.logger.level_*`.
         See :ref:`verbosity-levels` for more info.
-    output_type : {'input', 'cudf', 'cupy', 'numpy', 'numba'}, default=None
-        Variable to control output type of the results and attributes of
-        the estimator. If None, it'll inherit the output type set at the
-        module level, `cuml.global_output_type`.
-        See :ref:`output-data-type-configuration` for more info.
     """
     def __init__(self, client=None, streams_per_handle=0,
-                 verbose=False, **kwargs):
+                 verbose=False, batch_size=1024, **kwargs):
         super(KNeighborsClassifier, self).__init__(client=client,
                                                    verbose=verbose,
                                                    **kwargs)
         self.streams_per_handle = streams_per_handle
+        self.batch_size = batch_size
```

Comment:
Rather than setting each of these values on the instance and having to propagate them through the model creation functions, can we just append them to kwargs? That way we get the benefit of the docs but with less opportunity for missing an update during a refactor. For example, the batch size should be on the kwargs already in `_func_create_model` but if it's not, we can just pull the default from the constant defined in `NearestNeighbors`. What do you think?